crates/api/proto/fetch.proto

```diff
@@ -4,14 +4,33 @@ package kitsune2.fetch;
 
 import "op_store.proto";
 
-// A list of op ids.
-message OpIds {
+// A fetch request.
+message Request {
   // Op ids.
-  repeated bytes data = 1;
+  repeated bytes op_ids = 1;
 }
 
-// A list of ops.
-message Ops {
+// A fetch response.
+message Response {
   // Ops.
-  repeated kitsune2.op_store.Op op_list = 1;
+  repeated kitsune2.op_store.Op ops = 1;
+}
+
+// Fetch message.
+message K2FetchProto {
+  // Fetch message type.
+  enum FetchMessageType {
+    // Default value.
+    UNSPECIFIED = 0;
```

Comment:
This is recommended in [the language guide](https://protobuf.dev/programming-guides/proto3/#enum-default)